composer.json

```diff
@@ -56,18 +56,17 @@
         "craue/config-bundle": "^2.3.0",
         "defuse/php-encryption": "^2.1",
         "doctrine/collections": "^1.6",
-        "doctrine/common": "^2.13",
-        "doctrine/dbal": "^2.13",
-        "doctrine/doctrine-bundle": "^1.9",
-        "doctrine/doctrine-cache-bundle": "^1.3",
-        "doctrine/doctrine-migrations-bundle": "^1.3",
+        "doctrine/common": "^3.0",
+        "doctrine/dbal": "^3.3",
+        "doctrine/doctrine-bundle": "^2.0",
+        "doctrine/doctrine-migrations-bundle": "^3.2",
         "doctrine/event-manager": "^1.1",
-        "doctrine/migrations": "^1.8",
+        "doctrine/migrations": "^3.2",
         "doctrine/orm": "^2.6",
-        "doctrine/persistence": "^1.3",
+        "doctrine/persistence": "^3.0",
         "enshrined/svg-sanitize": "^0.15.4",
         "friendsofsymfony/jsrouting-bundle": "^2.2",
-        "friendsofsymfony/oauth-server-bundle": "^1.5",
+        "friendsofsymfony/oauth-server-bundle": "dev-master",
```

Comment:
Not a big fan of putting this kind of version constraint, could we maybe fix a commit for safety?

Reply:
Good idea, I'll lock to the latest current commit